builder/common/step_run_source_instance.go

```diff
@@ -377,6 +384,18 @@ func (s *StepRunSourceInstance) Run(ctx context.Context, state multistep.StateBa
 			return multistep.ActionHalt
 		}
 
+		if len(ec2Tags) > 0 {
+			for _, networkInterface := range instance.NetworkInterfaces {
+				_, err := ec2conn.CreateTags(&ec2.CreateTagsInput{
+					Tags:      ec2Tags,
+					Resources: []*string{networkInterface.NetworkInterfaceId},
+				})
+				log.Printf("Tagging network interface %s", *networkInterface.NetworkInterfaceId)
+				if err != nil {
+					ui.Say(fmt.Sprintf("Error tagging source instance's network interface: %s, skip tagging network interfaces", err))
```

Comment:
I think this should be a silent error to not make any breaking changes here. So we can print the error and get some feedback if someone got this error at all.

Reply:
I don't think this qualifies as silent error, but I agree that we shouldn't halt the build process if that happens for whatever reason.
I would maybe suggest we use `ui.Error` rather than `ui.Say` for reporting this to the user though

Reply:
About the message though, I would maybe suggest rephrasing it a little bit so users know which interface failed to be tagged
```suggestion
					ui.Say(fmt.Sprintf("Error tagging source instance's network interface %q: %s", *networkInterface.NetworkInterfaceId, err))
```